src/test/test.cpp

```diff
@@ -512,12 +512,6 @@ for_versions(std::vector<uint32> const& versions, Application& app,
 {
     for (auto v : versions)
```

Comment:
This is better, but we could still produce a cleaner implementation. How about just
```c++
if (std::find(versions.begin(), versions.end(), gTestingVersion) != versions.end())
{
    // Do the SECTION
}
```
with no `for` loop, no `continue`, no `return`? Just super simple, idiomatic, easy to read standard library functionality. The same applies for the other overload.

Reply:
Yeah good idea. Done.